R/qenv-eval_code.R

```diff
@@ -24,8 +24,12 @@ setMethod("eval_code", signature = c("qenv", "expression"), function(object, cod
 
   object@id <- c(object@id, id)
   object@env <- rlang::env_clone(object@env, parent = parent.env(.GlobalEnv))
+  object@code_dependency <- bind_code_dependency(object@code_dependency, code_dependency(code, object@env))
```

Comment:
Isn't this easier to grasp if for each `@code` element we have one `@code_dependency` element. I don't like the idea of `bind_code_dependency > bind_lists` in `eval_code`. Can we bind these lists in `get_code > get_code_dependency` instead?

Reply:
Sure, I think it's more transparent now. Just moved it to `get_code_dependencies` 4a326aadf04ab8bef06555102aa9fa80e81c62d2